### PR TITLE
[wip] Syndrones have nuke ops access, syndicomms

### PR DESCRIPTION
**Pull request by coiax**
- Minor cleaning of encryptionkey.dm and headset.dm
- Drones now can have default radios that spawns inside of them, which
  is deleted when they are destroyed (like their ID card)
- Removes syndrone cannibalism because current implementation is bugged because they can just repair themselves with a screwdriver anyway

:cl: coiax
rscadd: Syndrones now have Syndicate access on their internal ID cards, along with access to Syndicate communications.
/:cl:
- [ ] Test to see if they can talk and listen on their radios

### CODE REVIEW

**Comment by optimumtact** [comment]
What on earth is a syndrone?

**Comment by RemieRichards** [comment]
a syndrone is a small black and red drone with syndicate laws, they've existed about as long as normal drones have.

there's also the "badass syndrone" that spawns with an uplink and nuke op helmet.

**Comment by optimumtact** [comment]
Are they something explicitly spawned by the syndie team? If so then this is a fine change.

**Comment by RemieRichards** [comment]
admin only, but sometimes used in TC trades (traitor OR ops)
(also I was wrong both syndrones have TC, but where the normal syndrone has 10 the badass has 30 and a weapon auth implant)

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 1]
```diff
@@ -14,8 +14,8 @@
 	icon_state = "drone_synd"
 	icon_living = "drone_synd"
 	picked = TRUE //the appearence of syndrones is static, you don't get to change it.
-	health = 30
-	maxHealth = 120 //If you murder other drones and cannibalize them you can get much stronger
+	health = 50
+	maxHealth = 50
```
Comment (JJRcop) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: :rotating_light: :exclamation: :rotating_light: :rotating_light: :exclamation: :exclamation: :rotating_light: :exclamation: :exclamation: :rotating_light: :rotating_light: :exclamation: :rotating_light:
### __UNDOCUMENTED CHANGE__
:rotating_light: :exclamation: :rotating_light: :rotating_light: :exclamation: :exclamation: :rotating_light: :exclamation: :exclamation: :rotating_light: :rotating_light: :exclamation: :rotating_light:

Comment (coiax) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: oh yeah

Comment (JJRcop) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: also sorry it used to work but now it doesn't, guess it's okay to see it gone

**Comment by Shadowlight213** [comment]
@remierichards
Main difference isn't really the tc, it's that the regular one has a traitor uplink and the badass one has a nuke ops uplink.

**Comment by optimumtact** [comment]
ready?

**Comment by KorPhaeron** [comment]
stale

**Pull request closed without merging**